k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -295,6 +298,13 @@ public void populate(final Message message, final Account account) {
             mContactBadge.setVisibility(View.GONE);
         }
 
+        if (shouldShowSender(message)) {
+            mSenderView.setVisibility(VISIBLE);
```

Comment:
It is invisible in the xml file, but what will happen when the `populate` method is called twice? Methods that fill in view data should produce a consistent state generally (even though it might not be a problem for some particular instance)